src/pages/Notifications/EventLog/EventLogPage.tsx

```diff
@@ -123,18 +123,19 @@ export const EventLogPage: React.FunctionComponent = () => {
     async (id: UUID) => {
       const query = getEndpoint.query;
       const endpoint = await query(id);
-      if (endpoint.payload?.type === 'Endpoint') {
+      if (endpoint.payload?.type === 'EndpointDTO') {
         const type = endpoint.payload.value.type;
         switch (type) {
           case 'camel':
           case 'webhook':
           case 'ansible':
+          case 'pagerduty':
             return endpoint.payload.value.name;
           case 'email_subscription':
           case 'drawer': {
             const properties = endpoint.payload.value
-              .properties as Schemas.EmailSubscriptionProperties;
-            if (properties.only_admins) {
+              .properties as Schemas.SystemSubscriptionPropertiesDTO;
```

Comment:
Do we need DTO in here?
```suggestion
              .properties as Schemas.SystemSubscriptionProperties;
```